pkg/database/migrations.go

```diff
@@ -1929,6 +1929,17 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`ALTER TABLE authorized_app_stats DROP COLUMN IF EXISTS tokens_invalid`)
 			},
 		},
+		{
+			ID: "00083-AddTwilioMessageSid",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE sms_configs ADD COLUMN IF NOT EXISTS twilio_messaging_service_sid varchar(255)`)
```

Comment:
Sorry, I was trying to say: I don't think we want a new field/column at all.

- Make sure `from` can handle 255 characters
- Add something in sms.go:
    ```go
    if strings.HasPrefix(from, "MG") {
      q.Add("MessagingServiceSid", from)
    } else {
      q.Add("From", from)
    }
    ```

That should dramatically reduce complexity and avoid boolean logic for `if msid else from`.

Reply:
sure - expanded the existing db column instead.